[Bulk] Validation - Missing descriptions (#1451)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/controls/how-to-expose-properties-of-constituent-controls.md

---
title: "How to: Expose Properties of Constituent Controls"

> description: Learn how to expose the properties of Constituent Controls so that these controls that are usually private are available to future users.

Reviewer: ```suggestion
description: Learn how to expose the properties of Constituent Controls, controls that are usually private, can be made available for public access.
```
